### DISCUSSION TITLE
Is $nestedOperations still in the plans for Prisma extensions? Particularly for row level security.

### DISCUSSION THREAD

**Discussion by davepreston** [prisma/prisma#23727] [Q&A]
## Problem

I would like to make alterations to the where clause of every prisma query that runs to enforce row level security. I was able to do that at the top-level using extensions, but select/includes of relationships make it very difficult to identify existing or potential "where" clauses that need to be modified.

## Suggested solution

I'd like some way to alter all where clauses. I would expect something like a function that would be called with every where clause (even empty ones). I would tweak them as needed and return the altered where clause.

## Alternatives

A proposal for clientExtensions was discussed in October 2022 (https://github.com/prisma/prisma/issues/15074#issuecomment-1282419200). It referenced $nestedOperations. I believe that would meet my needs, but I couldn't find any reference to $nestedOperations after that point.

**Comment by ludralph** [comment]
Hi @davepreston 👋 

Thank you for raising this question.

$nestedOperation is still on our list to build one day, but we have no ETA at the moment.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
